swatch-system-conduit/src/test/java/org/candlepin/subscriptions/conduit/admin/InternalOrganizationSyncResourceTest.java

```diff
@@ -68,4 +70,20 @@ void syncOrgShouldThrowInternalServerError() throws Exception {
   void syncAllOrgsShouldReturnSuccess() {
     assertEquals("Success", resource.syncFullOrgList().getStatus());
   }
+
+  @Test
+  void hasOrgInSyncListShouldReturnTrue() {
+    when(repo.existsById("123")).thenReturn(true);
+    assertTrue(
+        "Org ID expected to exist but does not",
```

Comment:
Shouldn't the assertTrue message be "Org Id exists" instead of "does not"?

Reply:
This message is displayed only if the assertion fails, so if it does not exist then this message is displayed.

Reply:
Usually the assertion shouldn't fail which means in this case it should be always true. May be, I am missing something here.

Reply:
@kartikshahc the 'message' field only gets displayed when an assertion failed. It shouldn't fail but if something in the code changes and now it does fail this message will explain why the test failed.

Reply:
Ah ok! Makes sense will resolve these comments in that case.